src/main/java/org/weasis/dicom/google/api/GoogleAPIClient.java

```diff
@@ -358,6 +374,17 @@ public static String formatQuery(StudyQuery query) {
             );
         }
 
+        int pageNumber = query.getPage();
+        int pageSize = query.getPageSize();
+
+        if (pageSize > 0) {
```

Comment:
not sure if we should do it
default 5000 for limit is kind of bad for this swing ui so it's becomes slow
in current state this part can't be called with no limit, but in future isn't better for caller to set explicitly how many objects is going to be requested?

Reply:
Ah ok makes sense. Just wondering, should we have a max_limit (which will not slow swing UI) then? What if the user does not understand that it can slow the UI and passes a big value?

Reply:
>Just wondering, should we have a max_limit  
i think we shouldn't because goal is not limit user but make it clear that size of page directly affect UI
> What if the user does not understand that it can slow the UI and passes a big value?
well, i don't know how to make it more obvious

Reply:
I think we can atleast add a note in the documentation saying if the pageSize is too high it will slow the UI down.

Reply:
https://github.com/GoogleCloudPlatform/weasis-chcapi-extension/pull/17/commits/ab40281343e0c9b85a4e7b24df3e72eab4a10984#diff-436d0eb6316e4c426b0598c2387d693bR92
done